Reason: (e) sulu/skeleton 3.0 links the Sulu 2.6 docs of the markup bundle.
Evidence: https://docs.sulu.io/3.x/bundles/markup/index.html answers 200 without a redirect (curl, 2026-10-04).

--- a/config/packages/sulu_markup.yaml
+++ b/config/packages/sulu_markup.yaml
@@ -1,4 +1,4 @@
-# see also: https://docs.sulu.io/en/2.6/bundles/markup/index.html
+# see also: https://docs.sulu.io/3.x/bundles/markup/index.html
 sulu_markup:
     link_tag:
         provider_attribute: 'data-provider'
